cli/flags.rs

```diff
@@ -163,6 +163,7 @@ pub struct RunFlags {
 
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
 pub struct TaskFlags {
+  pub cwd: Option<String>,
```

Comment:
Consider using `Option<PathBuf>` instead?

Reply:
The config flag uses `Option<String>` 🤷‍♂️